aws/cf_template.go

```diff
@@ -24,6 +24,8 @@ const (
 	//
 	// [0]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenerrule.html#cfn-elasticloadbalancingv2-listenerrule-priority
 	internalTrafficDenyRulePriority int64 = 1
+
+	loadBalancerResourceLogicalID = "LB"
```

Comment:
add a comment that a change of this name will trigger either update or recreate (I am not sure) of all CF stacks owned by the controller in an AWS account.